upstream/CVE-2021-25741.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2021-25741",
+  "modified": "2021-09-13T20:58:56Z",
+  "published": "2021-09-13T20:58:56Z",
+  "summary": "Symlink Exchange Can Allow Host Filesystem Access",
+  "details": "A security issue was discovered in Kubernetes where a user may be able to create a container with subpath volume mounts to access files \u0026 directories outside of the volume, including on the host filesystem.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/kubelet"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.19.0"
+            },
+            {
+              "last_affected": "1.19.14"
+            }
```

Comment:
there seems to be a mistake in the [issue](https://github.com/kubernetes/kubernetes/issues/104980) and here.
I can't find commit for v1.19.

Reply:
removed this in https://github.com/aquasecurity/vuln-list-k8s/pull/87/commits/b912ab678b3386f8a6be8e8739eee2f421f596bc

Also added `reference` to this PR.

Reply:
My bad. I found this PR - https://github.com/kubernetes/kubernetes/pull/105058

```bash
➜  kubernetes git:(master) git tag --contains 7b343ec8ff77bfb17adc98658f4efc0b9b6970c9 | sort -V | head -n 2
v1.19.16
v1.19.17-rc.0
```

Updated in https://github.com/aquasecurity/vuln-list-k8s/pull/87/commits/fda45c0f7d8ad06524bb6a2c001742d3e57d0c4b